recipe/meta.yaml

```diff
@@ -11,47 +11,68 @@ source:
   sha256: 4fd2eb3736cea503be94ab18d31a669addbaf2dd6a8249527cef94163bb93975
 
 build:
+  number: 4
   noarch: python
-  number: 3
-  script: python -m pip install . -vv
 
-requirements:
-  host:
-    - hatchling
-    - pip
-    - python {{ python_min }}
+outputs:
+  - name: osmnx-base
+    script: build_base.sh
+    build:
+      noarch: python
+    requirements:
+      build:
+        - python                              # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
```

Comment:
Is this necessary if the package is pure Python? Why would we need `cross-python_` anything?

Reply:
I am not sure, I copied this line from GeoPandas receipt.

Reply:
You probably don't need. If I recall correctly we needed that for a workaround at some point.

Reply:
@ocefpaf Right, I removed it and it worked.